### DISCUSSION TITLE
Cant see Gitlab repositories in Vercel

### DISCUSSION THREAD

**Discussion by mohrazzak** [vercel/community#3651] [Help]
### Summary

I gave the access asked to Gitlab but no repositories were shown
this is the error given by Vercel

Error:
No Git Repositories Found

Try selecting a different Git account or organization on the top left.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/86966092/eeaa4a2e-607f-4232-ad9c-f8ff734802e9)

**Comment by mohrazzak** [comment]
![image](https://github.com/vercel/community/assets/86966092/028fcf4c-562c-4fa2-a5ab-e8861dd0ac0a)
there is no options

**Comment by amyegan** [comment]
Thanks for sharing this. It looks like GitLab has an active incident affecting our ability to import repos https://status.gitlab.com/pages/incident/5b36dc6502d06804c08349f7/64dd7791e88a000536314396

**Reply by mohrazzak** [reply]
So can you provide me more details 
Will it be solved soon within two days? or what exactly

**Reply by amyegan** [reply]
We do not have control over GitLab's servers, so we are not able to provide estimates or solve outages related to GitLab systems. However, the incident has been marked as [`resolved` on their status page](https://status.gitlab.com/pages/incident/5b36dc6502d06804c08349f7/64dd7791e88a000536314396) at this point. 

If you try to create a new deployment now, do you still have the same problem?

**Comment by mohrazzak** [comment]
No it's not working, the same problem occurred

‫في الخميس، 17 أغسطس 2023 في 8:30 م تمت كتابة ما يلي بواسطة ‪Amy‬‏ <‪
***@***.***‬‏>:‬

**Reply by amyegan** [reply]
Thanks for checking. Do you have [**Maintainer** access](https://docs.gitlab.com/ee/user/project/members/) to the repository? This is required so we can configure a webhook to automatically deploy pushed commits for you.

**Comment by mohrazzak** [comment]
No, I do not.
I will find out and reach out to you soon.

**Comment by MarcGaladrim** [comment]
Hello, I did encounter the same issue with the Maintainer access! Even with owner access

**Reply by mohrazzak** [reply]
@MarcGaladrim Fork it as a temporary fix
